core/arch/riscv64/riscv64.asm

```diff
@@ -322,8 +322,20 @@ GLOBAL_LABEL(back_from_native:)
 # if !defined(STANDALONE_UNIT_TEST) && !defined(STATIC_LIBRARY)
         DECLARE_FUNC(_start)
 GLOBAL_LABEL(_start:)
-/* FIXME i#3544: Not implemented */
-        ret
+        nop
+        mv      fp, x0  /* clear frame ptr for stack trace bottom */
+
+        CALLC3(GLOBAL_REF(relocate_dynamorio), 0, 0, sp)
+        mv      ARG2, x0
+        mv      ARG3, x0
+
+        /* Clear 2nd & 3rd args to distinguish from xfer_to_new_libdr */
+.L_start_invoke_C:
```

Comment:
Should be GLOBAL_LABEL since referenced outside of the function?

Reply:
Seems AArch64 doesn't have GLOBAL_LABEL. https://github.com/DynamoRIO/dynamorio/blob/master/core/arch/aarchxx/aarchxx.asm#L59.

Reply:
It should, since the assembler on Mac requires an _ and so someone will have to go add GLOBAL_LABEL to get aarch64 to work on Mac.  Let's use the GLOBAL_LABEL for RISCV now from the start to avoid similar problems and make it easier to port to other OS's and assemblers.